containers/btc-node/bitcoin.conf

```diff
@@ -1,11 +1,19 @@
 server=1
 regtest=1
 txindex=1
-fallbackfee=0.0000002
-maxconnections=1024
```

Comment:
This line was required to support configurations with a large number of clients.